ports/psoc6/mpconfigport.h

```diff
@@ -57,9 +46,10 @@
 // MicroPython emitters
 #define MICROPY_PERSISTENT_CODE_LOAD            (1)
 #define MICROPY_EMIT_THUMB                      (1)
-#define MICROPY_EMIT_THUMB_ARMV7M               (0)
+#define MICROPY_EMIT_THUMB_ARMV7M               (1)
 #define MICROPY_EMIT_INLINE_THUMB               (1)
-#define MICROPY_EMIT_INLINE_THUMB_FLOAT         (0)
+#define MICROPY_EMIT_INLINE_THUMB_FLOAT         (1)
+#define MICROPY_EMIT_ARM                        (0)
 
 // Optimisations
 // #define MICROPY_OPT_COMPUTED_GOTO               (1)
```

Comment:
??

Reply:
Port specific switches to be able to use inline assembler code and run the corresponding tests.

Reply:
Can there be some explanation on when to comment or uncomment this. Otherwise, if anything that is not currently required, i would just removed the commented code.

Reply:
Removed commented out lines